src/plugins/paced-transcript/index.js

```diff
@@ -13,9 +13,13 @@ function pacedTranscript(config) {
       source.publicId(),
       extendCloudinaryConfig(player.cloudinary.cloudinaryConfig(), { resource_type: 'raw' }),
     ) + '.transcript',
-    maxWords: config.maxWords || 5 // Number of words per caption
+    maxWords: config.maxWords || 5, // Number of words per caption
```

Comment:
No that's actually a good comment, we shouldn't have any default value set here :)

On the other hand, you'll still see some defaults being set here, i.e. `label`, you can look at it as plugin defaults - because from player perspective there is no default caption or caption label, but from the plugin perspective, when it is enabled, you do need to always have a label, so we can maybe write it more elegantly, but it will still need to be there somehow.